docker-compose.yml

```diff
@@ -11,33 +11,33 @@ services:
       - "5432:5432"
     volumes:
       - "dispatch-postgres:/var/lib/postgresql/data"
-  web:
-    <<: *restart_policy
+  core:
+    image: dispatch-local
+    env_file:
+      - .env
     build:
       # Pro-tip: point this to a relative directory containing the Dispatch
       # project root to pick up changes from your dev environment
       # (e.g., ../dispatch-root)
-      context: https://github.com/Netflix/dispatch.git#v20201001
-    image: dispatch-local-web
+      #context: https://github.com/Netflix/dispatch.git#v20201001
```

Comment:
nit: do we need to remove this commented line?

Reply:
Yes, in fact I need to reset that whole context.